src/components/IconSet/icons/index.ts

```diff
@@ -0,0 +1,5 @@
+export { default as chat } from './chat.png'
+export { default as discord } from './discord.png'
```

Comment:
did we remove them from dvc.org static? do we need those still?

Reply:
Good catch! Removed these icons from `static/` folder

Reply:
please, check other places assets we move as well (to the extent it is possible)